lib/electric_slide/call_queue.rb

```diff
@@ -241,6 +241,8 @@ def remove_agent(agent, extra_params = {})
     # Checks to see if any callers are waiting for an agent and attempts to connect them to
     # an available agent
     def check_for_connections
+      # Ensure there are no nil objects in the call queue before trying to connect
+      @queue.compact!
```

Comment:
Is there any theory as to how objects in the queue became `nil`? This seems like a bandaid, and probably incomplete?